ee_plugin/ee_plugin.py

```diff
@@ -87,28 +87,31 @@ def initGui(self):
         """Initialize the plugin GUI."""
         # Build actions
         ee_user_guide_action = QtWidgets.QAction(
-            icon("earth-engine.svg"),
-            self.tr("User Guide"),
+            icon=icon("earth-engine.svg"),
+            text=self.tr("User Guide"),
+            parent=self.iface.mainWindow(),
             triggered=self._run_cmd_ee_user_guide,
         )
         sign_in_action = QtWidgets.QAction(
-            icon("google-cloud.svg"),
-            self.tr("Sign-in"),
+            icon=icon("google-cloud.svg"),
+            text=self.tr("Sign-in"),
+            parent=self.iface.mainWindow(),
             triggered=self._run_cmd_sign_in,
         )
         set_cloud_project_action = QtWidgets.QAction(
-            icon("google-cloud-project.svg"),
-            self.tr("Set Project"),
+            icon=icon("google-cloud-project.svg"),
+            text=self.tr(f"Set Project"),
+            parent=self.iface.mainWindow(),
```

Comment:
@zacdezgeo the missing icons we experienced during a pair call was due to the missing `parent`. This PR should now be good to go!